lib/model.js

```diff
@@ -1569,6 +1569,7 @@ Model.prototype.count = function(options) {
   options.limit = null;
   options.offset = null;
   options.order = null;
+  options.attributes = [];
```

Comment:
Maybe this should be moved to `aggregate` if we apply scope to all aggregate calls now, what do you think @janmeier?